resources/build/build-utils.sh

```diff
@@ -270,21 +270,31 @@ replaceVersionStrings_Mkver() {
 # Standard build script functions for managing command line, actions and targets
 ################################################################################
 
-# TODO: colors are defined here and in shellHelperFunctions.sh
 # The following allows coloring of warning and error lines, but only works if there's a
 # terminal attached, so not on the build machine.
-if [[ -n "$TERM" ]] && [[ "$TERM" != "dumb" ]] && [[ "$TERM" != "unknown" ]]; then
+builder_use_color() {
```

Comment:
I don't think this method is intended to be used outside of this file, is it?

```suggestion
_builder_use_color() {
```